themes-default/slim/views/config_backuprestore.mako

```diff
@@ -97,7 +81,8 @@ const startVue = () => {
                         <div class="field-pair">
                             Select the folder you wish to save your backup file to:
                             <br><br>
-                            <input type="text" name="backupDir" id="backupDir" class="form-control input-sm input350"/>
+                            <!-- <input v-model="backup.dir" type="text" name="backupDir" id="backupDir" class="form-control input-sm input350"/>-->
+                            <file-browser name="backupDir" ref="backupDirBrowser" id="backupDir" title="Select Show Location" :initial-dir="backup.dir" @update:location="backup.dir = $event"></file-browser>
```

Comment:
1. I don't remember why I added the `ref=` in editShow but it's not needed.
`id=` is not needed here either.
2. `initial-dir` is not needed in this case, as it will always be empty (no config variable.
3. You need to add `local-storage-key="backupDir"`

Reply:
1 and 2 i get. But why do i need local storage key here?